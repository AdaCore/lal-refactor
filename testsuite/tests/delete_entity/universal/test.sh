cp ./main.orig ./main.adb
delete_entity -P default.gpr -S main.adb -L 2 -R 12
echo "--  (main.orig) -L 2 -R 12"
cat ./main.adb
