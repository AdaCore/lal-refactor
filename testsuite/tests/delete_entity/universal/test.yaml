title: delete_any

description: delete_enity tool test

driver: shell_script
