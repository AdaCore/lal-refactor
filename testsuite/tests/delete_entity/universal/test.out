--  (main.orig) -L 2 -R 12
procedure Main is

begin
   null;
end Main;
